usaspending_api/api_contracts/contracts/v2/agency/toptier_code/def_code.md

```diff
@@ -0,0 +1,111 @@
+
+Returns a list of Disaster Emergency Fund Codes in the Agency's appropriations for a single fiscal year
+
+## GET
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 086 (required, number)
+            The toptier code of an agency (could be a CGAC or FREC) so only numeric character strings of length 3-4 are accepted.
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY.
+        + `filter` (optional, string)
+            This will filter the Disaster Emergency Fund Codes by their code or title to those matching the text.
+        + `order` (optional, enum[string])
+            Indicates what direction results should be sorted by. Valid options include asc for ascending order or desc for descending order.
+            + Default: `desc`
+            + Members
+                + `desc`
+                + `asc`
+        + `sort` (optional, enum[string])
+            Optional parameter indicating what value results should be sorted by.
+            + Default: `obligated_amount`
+            + Members
+                + `name`
+                + `total_budgetary_resources`
+                + `obligated_amount`
+                + `gross_outlay_amount`
+        + `page` (optional, number)
+            The page number that is currently returned.
+            + Default: 1
+        + `limit` (optional, number)
+            How many results are returned.
+            + Default: 10
+
++ Response 200 (application/json)
+    + Attributes
+        + `toptier_code` (required, string)
+        + `fiscal_year` (required, number)
+        + `page_metadata` (required, PageMetadata, fixed-type)
+            Information used for pagination of results.
+        + `combined_total_budgetary_resources` (required, number)
+            Total Budgetary Resources associated with any Disaster Emergency Fund Code
+        + `combined_obligated_amount` (required, number)
+            Total Obligations associated with any Disaster Emergency Fund Code
+        + `combined_gross_outlay_amount` (required, number)
+            Total Outlays associated with any Disaster Emergency Fund Code
+        + `results` (required, array[DisasterEmergencyFundCode], fixed-type)
+        + `messages` (required, array[string], fixed-type)
+            An array of warnings or instructional directives to aid consumers of this endpoint with development and debugging.
+
+    + Body
+
+            {
+                "page_metadata": {
+                    "page": 1,
+                    "total": 1,
+                    "limit": 2,
+                    "next": 2,
+                    "previous": null,
+                    "hasNext": true,
+                    "hasPrevious": false,
+                },
+                "totals": {
```

Comment:
The response above has these values at the top-level (and using the `combined` naming convention) instead of nested under `totals`